mini-wallet.md

```diff
@@ -36,30 +29,35 @@ dmw start-server --help
 Usage: dmw start-server [OPTIONS]
 
 Options:
-  -n, --name TEXT                 Application name.  [default: mini-wallet]
-  -h, --host TEXT                 Start server host.  [default: localhost]
-  -p, --port INTEGER              Start server port.  [default: 8888]
-  -j, --jsonrpc TEXT              Diem fullnode JSON-RPC URL.  [default:
+-n, --name TEXT                 Application name.  [default: mini-wallet]
+-h, --host TEXT                 Start server host.  [default: localhost]
+-p, --port INTEGER              Start server port.  [default: 8888]
+-j, --jsonrpc TEXT              Diem fullnode JSON-RPC URL.  [default:
                                   http://testnet.diem.com/v1]
 
-  -f, --faucet TEXT               Testnet faucet URL.  [default:
-                                  http://testnet.diem.com/mint]
+-f, --faucet TEXT               Testnet faucet URL.  [default:
+                                 http://testnet.diem.com/mint]
 
-  -l, --logfile TEXT              Log to a file instead of printing into
+-l, --logfile TEXT              Log to a file instead of printing into
                                   console.
 
-  -o, --enable-debug-api BOOLEAN  Enable debug API.  [default: True]
-  --help                          Show this message and exit.
+-o, --enable-debug-api BOOLEAN  Enable debug API.  [default: True]
+--help                          Show this message and exit.
 ```
 
+View the DiemMiniWallet API specification (OpenAPI Specification 3.0.3) by opening http://localhost:8888. These specs include simple examples to try out the MiniWallet API. A hosted version of the API specification document is also available here.
 
-## MiniWallet Test Suite
 
-Try out the MiniWallet Test Suite by hitting the target server we started by `dmw start-server`
-```
-dmw test --target http://localhost:8888
-```
-You should see something like this:
+## Use the Diem MiniWallet Test Suite
+
+The Diem MiniWallet Test Suite is a set of tests built on top of the Diem MiniWallet API and is used to validate a wallet app’s integration with the Diem Payment Network (DPN). 
+ 
+Use the MiniWallet Test Suite to automate testing and checking if your wallet application meets our requirements. 
+
+1. Hit the target server we started by `dmw start-server`:
+`dmw test --target http://localhost:8888` 
+
+You should see the following response:
```

Comment:
`response` => `test report`

the following content is not response from a server, it is test report when running test.

Reply:
also the above `1. ...` will have left padding, but above line and the following code block will not have left padding, the rendered result looks odd.